### PR TITLE
current user in post comments

### PR DESCRIPTION
**Pull request by levity**

### CODE REVIEW

File: api/models/Post.js [hunk 1]
```diff
@@ -9,10 +9,17 @@ import { fulfillRequest, unfulfillRequest } from './post/request'
 import EnsureLoad from './mixins/EnsureLoad'
 import { countTotal } from '../../lib/util/knex'
 
-const commentersQuery = (limit, post) => q => {
+const commentersQuery = (limit, post, currentUserId) => q => {
   q.select('users.*', 'comments.user_id')
   q.join('comments', 'comments.user_id', 'users.id')
-  q.where('comments.post_id', '=', post.id)
+  if (currentUserId) {
```
Comment (levity) [inline api/models/Post.js]: To order by a specific user ID:

```js
q.orderBy(bookshelf.knex.raw(`case when user_id = ${id} then -1 else user_id end`)
```

Comment (levity) [inline api/models/Post.js]: @Connoropolous, to order by a specific user ID, do this:

```js
q.orderBy(bookshelf.knex.raw(`case when user_id = ${id} then -1 else user_id end`)
```

Comment (Connoropolous) [inline api/models/Post.js]: ah, wow, ok cool

**Comment by Connoropolous** [comment]
@levity ready for review

**Comment by levity** [comment]
i can't approve it because i created it :) so i'll just merge

**Pull request merged**